src/main/resources/react4xp/_entries/Dashboard/containers/DataQueries/DataQueryTable.jsx

```diff
@@ -18,12 +18,41 @@ export function DataQueryTable(props) {
   const contentStudioBaseUrl = useSelector(selectContentStudioBaseUrl)
   const io = useContext(WebSocketContext)
   const dispatch = useDispatch()
+  const [modalShow, setModalShow] = React.useState(false)
 
   function updateAll() {
     const ids = dataQueries.filter((q) => !q.loading).map((q) => q.id)
     requestDatasetUpdate(dispatch, io, ids)
   }
 
+  function ConfirmationModal(props) {
+    return (
+      <Modal
+        {...props}
+        size="lg"
+        aria-labelledby="contained-modal-title-vcenter"
+        centered
+      >
+        <Modal.Header closeButton>
+          <Modal.Title id="contained-modal-title-vcenter">
+              Oppdatere liste
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <h4>Er du sikker på at du vil oppdatere listen?</h4>
+          <p>
+              trykker du ja, vil du oppdatere alle tabeller og figurer i hele listen og
+              jobben kan ikke stoppes når den er blitt startet
+          </p>
+        </Modal.Body>
+        <Modal.Footer>
+          <Button variant="secondary" onClick={props.onHide}>Avbryt</Button>
```

Comment:
kanskje bytte til onClick={() => setModalShow(false)} , får en varsel på at onHide is missing in props validation